quickstart/WORKSPACE

```diff
@@ -12,29 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# This WORKSPACE verifies that the google-cloud-cpp-spanner libraries can be
-# loaded into a different Bazel project.
-workspace(name = "com_github_googleapis_google_cloud_cpp_spanner_test_install")
+"""This WORKSPACE shows how to use google-cloud-cpp-spanner from a larger Bazel project."""
 
-# To use the Cloud Spanner C++ client library applications should download it
-# using `http_archive()`. For example, use the following Starlark commands
-# (change the version and SHA256 hash as needed):
-#
-# load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-#
-# http_archive(
-#     name = "com_github_googleapis_google_cloud_cpp_spanner",
-#     sha256 = "a833d3c1a6d127132e961350829babac521b62b4c837b88d7c219b400e98fed1",
-#     strip_prefix = "google-cloud-cpp-spanner-0.8.0",
-#     url = "https://github.com/googleapis/google-cloud-cpp-spanner/archive/v0.8.0.tar.gz",
-# )
-#
-# In this WORKSPACE file we want to test that the *current* version is correct,
-# the CI builds always have the source in /v, so we use:
-#
-local_repository(
+workspace(name = "google_cloud_cpp_spanner_quickstart")
```

Comment:
We recently updated the [suggested `WORKSPACE` content here](https://github.com/googleapis/google-cloud-cpp-spanner/blob/master/INSTALL.md#using-google-cloud-cpp-spanner-in-bazel-based-projects), and it contains a few comments to explain to users what the rules are for. I think it would be useful to keep them, so you may want to replace this file w/ the content from [here](https://github.com/googleapis/google-cloud-cpp-spanner/blob/master/INSTALL.md#using-google-cloud-cpp-spanner-in-bazel-based-projects).

(keeping the `workspace` rule is fine, if you prefer)

Reply:
Made a couple of passes, let me know what you think.